index.rst

```diff
@@ -1,61 +1,106 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. sectnum::
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Abstract
+========
 
-  Use the following syntax for sections:
+Proprietary Rubin Observatory astronomical data must be transferred from the summit facility where the data is captured to the USDF (US Data Facility) for processing.
+This document proposes a data transfer implementation strategy to maintain confidentiality of the data in transit.
+It addresses the concerns identified in DMTN-108_.
 
-  Sections
-  ========
+.. _DMTN-108: https://dmtn-108.lsst.io/
 
-  and
+Problem statement
+=================
 
-  Subsections
-  -----------
+Data captured by the Charles Simonyi Survey Telescope on Cerro Pachón in Chile will be transferred from the summit facility (via dedicated private fiber) to a base facility in La Serena.
```

Comment:
Technically, data is captured by the camera (LSSTCam or ComCam) and not the telescope.

We are almost certainly going to change to a transfer process that directly goes from the summit to the USDF, without a TCP connection endpoint at the base facility, for nearly all prompt data.

Similarly, metadata (EFD and LFA contents) will be transferred directly from summit to USDF.

Reply:
Thanks, incorporated that and fixed the camera vs. telescope distinction. I left in the exception for data transfer to the base just in case, with a clarification that we don't intend to use it, but if you think it would just confuse matters, I can rip it out entirely.